build/roundProgress.js

```diff
@@ -1,4 +1,4 @@
-/* angular-svg-round-progressbar@0.4.6 2016-07-06 */
+/* angular-svg-round-progressbar@0.4.6 2016-08-16 */
```

Comment:
The above change was the result of running the grunt build task. The .gitignore file does not exclude the build/roundProgress.js file from any commits. Do you want me to manually exclude this change when performing a commit?